src/style.css

```diff
@@ -1,3 +1,185 @@
+    margin: 0;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
+    vertical-align: baseline;
+  }
+  /* HTML5 display-role reset for older browsers */
+  article, aside, details, figcaption, figure,
+  footer, header, hgroup, menu, nav, section {
+    display: block;
+  }
+  body {
+    line-height: 1;
+  }
+  ol, ul {
+    list-style: none;
+  }
+  blockquote, q {
+    quotes: none;
+  }
+  blockquote:before, blockquote:after,
+  q:before, q:after {
+    content: '';
+    content: none;
+  }
+  table {
+    border-collapse: collapse;
+    border-spacing: 0;
+  }
+/* #endregion CSS Reset  */
+/* #region RobotoFonts */
+.roboto-thin {
+  font-family: "Roboto", sans-serif;
+  font-weight: 100;
+  font-style: normal;
+}
+
+.roboto-light {
+  font-family: "Roboto", sans-serif;
+  font-weight: 300;
+  font-style: normal;
+}
+
+.roboto-regular {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.roboto-medium {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.roboto-bold {
+  font-family: "Roboto", sans-serif;
+  font-weight: 700;
+  font-style: normal;
+}
+
+.roboto-black {
+  font-family: "Roboto", sans-serif;
+  font-weight: 900;
+  font-style: normal;
+}
+
+.roboto-thin-italic {
+  font-family: "Roboto", sans-serif;
+  font-weight: 100;
+  font-style: italic;
+}
+
+.roboto-light-italic {
+  font-family: "Roboto", sans-serif;
+  font-weight: 300;
+  font-style: italic;
+}
+
+.roboto-regular-italic {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-style: italic;
+}
+
+.roboto-medium-italic {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-style: italic;
+}
+
+.roboto-bold-italic {
+  font-family: "Roboto", sans-serif;
+  font-weight: 700;
+  font-style: italic;
+}
+
+.roboto-black-italic {
+  font-family: "Roboto", sans-serif;
+  font-weight: 900;
+  font-style: italic;
+}
+/* #endregion RobbotoFonts */
+/* #region Header*/
+.header {
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  background-color: #FFFFFF;
+}
+
+.header__logo_url {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  background-color: #ffffff;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin-right: 50px;
+}
+
+.nav__url {
+  display: block;
+  color: #000000;
+  text-decoration: none;
+  font-family: Roboto;
+  font-size: 12px;
+  font-weight: 500;
+  text-align: left;
+  height: 60px;
+  line-height: 60px;
+  margin: 0 10px;
+}
+
+.nav__url:hover {
+  color: #00ACDC;
+}
+
+.is-active {
+  color: #00ACDC;
+  position: relative;
+  ::after
+
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  background-color: #00ACDC;
+  height: 4px;
+  /* ------------------------------------------- */
+  width: 37px;
+  /* Impossible to set same length in pixels as url-word length */
+  /* ------------------------------------------- */
```

Comment:
remove all comments

Reply:
Ok

Reply:
But what a solution to make a length same to url-word length without JavaScript?